ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -210,6 +210,34 @@
           --for=condition=ready
           --timeout={{ cifmw_edpm_prepare_timeout }}m
 
+    - name: Get CA bundle data
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.command:
+        cmd: 'oc get secret combined-ca-bundle -n openstack -o "jsonpath={.data.tls-ca-bundle\.pem}" --ignore-not-found'
+      register: ca_bundle_data
+
+    - name: Get CA bundle
+      when: ca_bundle_data.stdout | length > 0
+      ansible.builtin.set_fact:
+        ca_bundle: >-
+          {{ ca_bundle_data.stdout | ansible.builtin.b64decode }}
+
+    - name: Creating tls-ca-bundle.pem
+      when: ca_bundle | length > 0
+      ansible.builtin.copy:
+        dest: "{{ cifmw_edpm_prepare_basedir }}/tls-ca-bundle.pem"
```

Comment:
What are your thoughts on restricting permissions of this to a `600`?

Reply:
its the ca bundle which has no private key information. so there is no need to restrict permissions.